docs/products/mysql/concepts/mysql-replication.rst

```diff
@@ -0,0 +1,31 @@
+MySQL replication
+=================
+
+.. _replication-overview:
```

Comment:
Replication is a concept that we have in multiple services, I'll therefore add mysql to the ref
```suggestion
.. _myslq-replication-overview:
```

Reply:
it needs to be changed across all the files, so I will add a separated commit for it.